tests/unit/App.spec.js

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
+import App from '@/App.vue';
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+
+describe('App.vue', () => {
+  it('mounts and creates the app container', () => {
+    const store = new Vuex.Store({
+      state: { tasks: [] },
+      actions: { readTasks() {} },
+    });
+    const wrapper = shallowMount(App, { store, localVue });
+    expect(wrapper.find('#app').exists()).to.be.true;
```

Comment:
This seems so arcane to me: `to.be.true`. Welp.

Reply:
This is the API for behaviour driven tests in chai, there are some good explanations in the [introduction](https://www.chaijs.com/guide/styles/#expect) and [api docs](https://www.chaijs.com/api/bdd/)